paysley/readme.txt

```diff
@@ -5,7 +5,7 @@ Tags: credit card, paysley, google pay, apple pay, payment method, payment gatew
 Requires at least: 5.0
 Tested up to: 6.4.3
```

Comment:
@erwinesphere can we change this to latest wp version

Reply:
yes. we can. but it already tested or not?

Reply:
no its not apprently so we have to leave it as is till its teseted